Cargo.toml

```diff
@@ -26,7 +26,7 @@ r2d2_sqlite = "0.1.*"
 rocket = "0.2.*"
 rocket_codegen = "0.2.*"
 rocket_contrib = "0.2.*"
-rusqlite = "0.10.*"
+rusqlite = "0.11.*"
```

Comment:
Since rusqlite is something we use via 'extern crate' it should stay. If we have anything in there we don't use directly then I'd agree to removing them.